.github/workflows/ubuntubuilds.yml

```diff
@@ -23,19 +23,28 @@ jobs:
           submodules: recursive
 
       - name: Build
-        run:
-          mkdir build && cd build && cmake -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_DDS=false -DKPSR_WITH_ZMQ=false \
-          -DKPSR_TEST_PERFORMANCE=true -DKPSR_WITH_SOCKET=true -DKPSR_WITH_YAML=false -DKPSR_WITH_CODE_METRICS=true -DCMAKE_PREFIX_PATH=/opt/klepsydra/thirdparties \
-          -DCMAKE_BUILD_TYPE=Release .. && make -j ${nproc}
+        run: |
+          mkdir build && cd build 
+          cmake -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_DDS=false -DKPSR_WITH_ZMQ=false -DKPSR_TEST_PERFORMANCE=true -DKPSR_WITH_SOCKET=true -DKPSR_WITH_YAML=false -DKPSR_WITH_CODE_METRICS=true -DCMAKE_PREFIX_PATH=/opt/klepsydra/thirdparties -DCMAKE_BUILD_TYPE=Release ../ 
+          make -j ${nproc}
 
       - name: Test
         run: cd build && make test
```

Comment:
Based on some test failures I've got, it might be interesting to run tests in random order. This can be done by `make test ARGS="--gtest_shuffle"`

Reply:
Ok. I will add it.